src/client/oicp/CpoOICPClient.ts

```diff
@@ -0,0 +1,1272 @@
+        sendResult.logs.push(
+          `Pushed/Updated successfully EVSEs from tenant '${this.tenant.id}'`
+        );
+      } catch (error) {
+        sendResult.failure = sendResult.total;
+        sendResult.chargeBoxIDsInFailure.push(...chargeBoxIDsToProcessFromInput);
+        sendResult.logs.push(
+          `Failed to update the EVSEs from tenant '${this.tenant.id}': ${String(error.message)}`
+        );
+      }
+      // If (sendResult.failure > 0) {
+      //   // Send notification to admins
+      //   // eslint-disable-next-line @typescript-eslint/no-floating-promises
+      //   NotificationHandler.sendOICPPatchChargingStationsStatusesError(
+      //     this.tenant.id,
+      //     {
+      //       location: location.name,
+      //       evseDashboardURL: Utils.buildEvseURL((await TenantStorage.getTenant(this.tenant.id)).subdomain),
+      //     }
+      //   );
+      // }
+    }
+
+    // Log error if any
+    if (sendResult.failure > 0) {
+      // Log error if failure
+      Logging.logError({
+        tenantID: this.tenant.id,
+        action: ServerAction.OICP_PUSH_EVSES,
+        message: `Pushing of ${sendResult.failure} EVSEs has been done with errors (see details)`,
+        detailedMessages: { logs: sendResult.logs },
+        module: MODULE_NAME, method: 'sendEVSEs'
+      });
+    } else if (sendResult.success > 0) {
+      // Log info
+      Logging.logInfo({
+        tenantID: this.tenant.id,
+        action: ServerAction.OICP_PUSH_EVSES,
+        message: `Pushing of ${sendResult.success} EVSEs has been done successfully (see details)`,
+        detailedMessages: { logs: sendResult.logs },
+        module: MODULE_NAME, method: 'sendEVSEs'
+      });
+    }
+    // Save result in oicp endpoint
+    this.oicpEndpoint.lastPatchJobOn = startDate;
+    // Set result
+    if (sendResult) {
+      this.oicpEndpoint.lastPatchJobResult = {
+        successNbr: sendResult.success,
+        failureNbr: sendResult.failure,
+        totalNbr: sendResult.total,
+        chargeBoxIDsInFailure: _.uniq(sendResult.chargeBoxIDsInFailure),
+        chargeBoxIDsInSuccess: _.uniq(sendResult.chargeBoxIDsInSuccess)
+      };
+    } else {
+      this.oicpEndpoint.lastPatchJobResult = {
+        successNbr: 0,
+        failureNbr: 0,
+        totalNbr: 0,
+        chargeBoxIDsInFailure: [],
+        chargeBoxIDsInSuccess: []
+      };
+    }
+    // Save
+    await OICPEndpointStorage.saveOicpEndpoint(this.tenant.id, this.oicpEndpoint);
+    // Return result
+    return sendResult;
+  }
+
+  /**
+   * Send all EVSE Statuses
+   */
+  async sendEVSEStatuses(processAllEVSEs = true, actionType?: OICPActionType): Promise<OICPJobResult> {
+    if (!actionType) {
+      actionType = OICPActionType.fullLoad;
+    }
+    // Result
+    const sendResult = {
+      success: 0,
+      failure: 0,
+      total: 0,
+      logs: [],
+      chargeBoxIDsInFailure: [],
+      chargeBoxIDsInSuccess: []
```

Comment:
Same remark: OICPJobResult type does not contain that fields.

Reply:
ok, done